### PR TITLE
Update URL for DARS repository

### PR DESCRIPTION
**Pull request by bd-dean**

### CODE REVIEW

**Comment by bd-dean** [comment]
The DARS github repository is migrating from the clearlinux repo to the intel/stacks repo.  This reflects that change.

**Comment by luisfponce** [comment]
LGTM just one line to modify.
Started a review for it.

**Comment by mvincerx** [comment]
LGTM.  @luisfponce, please share your review. I don't see anything posted yet. Saludos.

File: source/guides/stacks/dars.rst [hunk 1]
```diff
@@ -102,13 +102,13 @@ To construct images with |CL|, start with a |CL| development platform that has t
 
    .. code-block:: bash
 
-      git clone https://github.com/clearlinux/dockerfiles.git
+      git clone https://github.com/intel/stacks
```
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/guides/stacks/dars.rst]: Good catch. Thanks.

**Comment by mvincerx** [comment]
@bd-dean, where are you on this doc? Seems we're really close. 
@luisfponce is your review complete?

File: source/guides/stacks/dars.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (bd-dean) [reply source/guides/stacks/dars.rst]: Added .git and changed the reference to CL Github repo to "Intel stacks" Github repo.  Should be good to go now.

[the remark this answers is not in the document]
Comment (mvincerx) [reply source/guides/stacks/dars.rst]: Thanks!

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**